themes/custom/lex/scss/molecules/_card_button.scss

```diff
@@ -0,0 +1,36 @@
+.card-button {
+    border: 1px solid #0558A7;
+    width: 250px;
+    background-color: #ffff;
+    color: #0558A7;
+
+    // img:before {
+    //     display: block;
+    //     position: absolute;
+    //     height: 100%;
+    //     width: 100%;
+    //     bottom: 0;
+    //     transition: height 0.5s ease-out;
+    //     background: linear-gradient(to bottom, trasnparent 0%, #E1B111 100%);
+    // }
```

Comment:
Remove this if we don't need it.

Reply:
Holding onto this while I look for a better way to do the image gradient.